src/components/Goals/BikeGoal/BikeGoalAlertSuccessSwitcher.jsx

```diff
@@ -24,14 +24,10 @@ const BikeGoalAlertSuccessSwitcher = ({ className }) => {
   const { isMobile } = useBreakpoints()
   const classes = useStyles()
 
-  const {
-    isLoading,
-    value: showAlertSuccess = true,
-    save: setShowAlertSuccess
-  } = useSettings('bikeGoal.showAlertSuccess')
+  const { isLoading, value, save } = useSettings('bikeGoal.showAlertSuccess')
```

Comment:
The issue with omitting the default value, is that the value will be `undefined` until you click on the switch.
And you will likely get a warning like this: "A component is changing an uncontrolled input to be controlled".
Also once you changed it in the settings via the admin setting, i don't think there is any obvious way to reset it so you can actually test in real conditions?

Reply:
Vu ensemble, on va laisse comme ça ici ça n'a pas de sens d'avoir une valeur par défaut sur ce swich, étant donnée qu'idéalement il faudrait la compute en fonction des trips de l'objectif vélo.

Je corrige par contre sur l'autre composant